### PR TITLE
feat(protocol-designer): test coverage to modals, FileSideBar, FilePage, EditModules

### PR DESCRIPTION
**Pull request by jerader**
partially closes RAUT-943

# Overview

Started adding some test coverage back to PD components since Enzyme was deprecated a few months ago so we lost a lot of PD test coverage. I will add more test coverage in a follow up PR.

# Test Plan

Just confirm that my tests make sense and the checks pass. I didn't alter any behavior in PD

# Changelog

- add a bunch of test coverage

# Review requests

see test plan

# Risk assessment

low

### CODE REVIEW

File: protocol-designer/src/components/FilePage.tsx [hunk 1]
```diff
@@ -30,23 +30,12 @@ import formStyles from '../components/forms/forms.css'
...
-
-export interface Props {
-  formValues: FileMetadataFields
-  instruments: React.ComponentProps<typeof InstrumentGroup>
-  goToNextPage: () => unknown
-  saveFileMetadata: (fileMetaDataFields: FileMetadataFields) => void
-  swapPipettes: () => unknown
-  modules: ModulesForEditModulesCard
-  t: any
-}
```
Comment (jerader) [inline protocol-designer/src/components/FilePage.tsx:43]: this wasn't being used at all. I forgot to remove it in my class/connect function PR

File: protocol-designer/src/components/FileSidebar/FileSidebar.tsx [hunk 2]
```diff
@@ -132,7 +132,7 @@ function getWarningContent({
     .map(moduleOnDeck => t(`modules:module_long_names.${moduleOnDeck.type}`))
     .join(' and ')
 
-  if (pipettesWithoutStep.length && modulesWithoutStep.length) {
+  if (pipettesWithoutStep.length > 0 && modulesWithoutStep.length > 0) {
```
Comment (mjhuff) [inline protocol-designer/src/components/FileSidebar/FileSidebar.tsx]: This is ok! Nit, I do think `pipettesWithoutStep.length` is fine or `Boolean(pipettesWithoutStep.length)` is the more common way to do this if you want to be very explicit.

**Review by mjhuff — APPROVED** [review]
Looks great! Left one non-blocking comment. Thanks for adding all these tests!

File: protocol-designer/src/components/FileSidebar/FileSidebar.tsx (hunk 2 above)
Comment (jerader) [reply protocol-designer/src/components/FileSidebar/FileSidebar.tsx]: thank you! for some reason i thought this was a bug the way it was before. I'll clean it up using `Boolean(pipettesWithoutStep.length)`

**Pull request merged**